File: HELLO_VANILLA/vehicle.js
class Vehicle{
	constructor(){
		this.speed = 0;
	}
	
	excel() {
		this.speed++;
	}
}
module.exports = Vehicle;

if (require.main === module) {
	// 사용 예시
	const ve = new Vehicle();
	console.log("speed:", ve.speed); 
	ve.excel();
	console.log("speed:", ve.speed);
}

File: HELLO_JAVA/src/Vehicle.java

public class Vehicle {
	int speed = 0;
	
	public void excel() {
		speed++;
	}

	public static void main(String[] args) {
		Vehicle ve = new Vehicle();
		System.out.println("speed:"+ve.speed);
		ve.excel();
		System.out.println("speed:"+ve.speed);
	}

}
